### ISSUE TITLE
Helm repo support

### ISSUE THREAD

**Issue by jcpunk** [pulp/pulpcore#3324]
**Is your feature request related to a problem? Please describe.**
When containers that are part of an existing Helm chart get synced down to Pulp they lose connection to their representative charts.

**Describe the solution you'd like**
Helm repo support

**Describe alternatives you've considered**
The single artifact repo didn't really work well for this task

**Additional context**
https://helm.sh/
https://helm.sh/docs/glossary/

**Comment by ipanova** [comment]
Do i read correctly this is a request to add helm chart repository served in its native format and not packaged as a container image served form a docker registry?

**Comment by jcpunk** [comment]
Yes, I'd love to have pulp able to provide a helm chart repository served in its native format.

**Comment by wombelix** [comment]
I would like to see that feature as well. That Helm OCI support was recently added is great but the classic way in addition would make it even more flexible.
Did someone already started working on it? If not I would like to give it a try. What's the process to get it as official plugin into the pulp GitHub organization as soon it's ready? Anything I have to consider or be aware of upfront?

Cheers
Dom

**Comment by lubosmj** [comment]
We added support to OCI-image-based helm charts to pulp_container recently. It has not been released yet.

A common workflow for establishing a new plugin includes generating the template from `plugin_template` and then starting utilizing some of the features provided by pulpcore. We can help you with that along the way.

https://docs.pulpproject.org/pulpcore/plugin_dev/plugin-writer/index.html
https://github.com/pulp/plugin_template
https://pulpproject.org/help/

**Comment by ipanova** [comment]
@wombelix the amount of docs might be overwhelming, I suggest to join #pulp-dev on matrix and ask questions along the way
